### PR TITLE
Fix README type error

### PR DESCRIPTION
**Pull request by hongyunyan**
### What problem does this PR solve?

Issue Number: ref #5178

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by hongyunyan** [comment]
cc @JaySon-Huang

File: README.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ To run micro benchmark tests, you need to build with -DCMAKE_BUILD_TYPE=RELEASE
```
Comment (JaySon-Huang) [inline README.md:261]: line 256 should be "under $WORKSPACE/tests dir"

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Review by SchrodingerZhu — APPROVED** [review]

**Pull request merged**